packages/caver-klay/caver-klay-accounts/src/accountKey/accountKeyMultiSig.js

```diff
@@ -0,0 +1,50 @@
+const AccountKeyEnum = require('./accountKeyEnum').AccountKeyEnum
+
+const MAX_MULTISIG_KEY_LENGTH = 10
+
+class AccountKeyMultiSig {
+	constructor(keys) {
+		if (keys instanceof AccountKeyMultiSig) keys = keys.keys
+
+		if (!Array.isArray(keys)) throw new Error('To create AccountKeyMultiSig, an array of private key strings is required.')
+		if (keys.length === 0) throw new Error(`Empty array.`)
+		if (keys.length > MAX_MULTISIG_KEY_LENGTH) throw new Error(`The maximum number of keys is ${MAX_MULTISIG_KEY_LENGTH}.`)
+		if (isDuple(keys)) throw new Error('There is a duplicate key.')
+		
+		this._keys = keys
+	}
+
+	get type() { return AccountKeyEnum.ACCOUNT_KEY_MULTISIG }
+
+	get defaultKey() { return this._keys[0] }
+	get keys() { return this._keys }
+	get transactionKey() { return this._keys }
+	get updateKey() { return this._keys }
+	get feePayerKey() { return this._keys }
+
+	toPublicKey(toPublicKeyFunc) {
+		const keys = []
```

Comment:
wrong indentation?

Reply:
Oh, in my IDE this looks fine, but here looks wrong indentation!! i will fix it :) Thank you.

Reply:
Hmmm.... I tried to fix indentation, but after remove one 'tap', this line will have same indent with L25.